### PR TITLE
feat(web): see start date has been set in the case timetable section …

### PR DESCRIPTION
**Pull request by kandcdev**
…of the case page

## Issue ticket number and link
https://pins-ds.atlassian.net/browse/BOAT-79

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/appeals/appeal-details/appeal-details.controller.js [hunk 1]
```diff
@@ -19,6 +20,8 @@ export const viewAppealDetails = async (request, response) => {
 			? Object.values(appealDetails?.appealSite)?.join(', ')
 			: 'Address not known';
 
+		const startedAt = appealDetails?.startedAt && dateFormatter(appealDetails.startedAt);
+
 		const formattedAppeal = {
```
Comment (fabfab-kc) [inline apps/web/src/server/appeals/appeal-details/appeal-details.controller.js:25]: I would probably move this to a mapper, where in the future we can add additional projections of the data for the upcoming screens.

File: apps/web/src/server/lib/date-formatter.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/**
+ * converts an ISO date string to Europe/London time zone date string
+ *
+ * @param {string} dateISOString
+ * @returns {string}
+ */
+export const dateFormatter = (dateISOString) =>
```
Comment (fabfab-kc) [inline apps/web/src/server/lib/date-formatter.js:7]: There is a simila function on the API app... worth discussing if we could consolidate in some shared helpers

**Review by fabfab-kc — APPROVED** [review]
Some comments to suggest improvement, to tackle in future refactorings...

**Comment by kandcdev** [comment]
Closed due changes done in https://github.com/Planning-Inspectorate/back-office/pull/944

**Pull request closed without merging**